examples/AspNetCore/README.md

```diff
@@ -13,6 +13,32 @@ service name, version and the machine on which this program is running.
 The sample rate is set to emit all the traces using `AlwaysOnSampler`.
 You can try out different samplers like `TraceIdRatioBasedSampler`.
 
+## Export metrics to Prometheus
+
+Switch to Prometheus by setting `UseMetricsExporter` to `"prometheus"` in `appsettings.json`.
+
+Follow this [guide](../../docs/metrics/getting-started-prometheus-grafana/README.md#collect-metrics-using-prometheus)
+on how to set up Prometheus.
+
+Note that for this example, the `yml` file configuration needs an additional
+parameter `scheme` for HTTPS. Here goes an example:
+
+```yaml
+global:
+  scrape_interval: 10s
+  evaluation_interval: 10s
+scrape_configs:
+  - job_name: "otel"
+    scheme: https
+    static_configs:
+      - targets: ["localhost:53750"]
+```
+
+See the metrics at [https://localhost:53750/metrics](https://localhost:53750/metrics).
+
+Make a request at [https://localhost:53750/WeatherForecast](https://localhost:53750/WeatherForecast)
```

Comment:
not sure what is the intention here? making a request to this path triggers metrics, yes. But it also triggers logs and traces as well..

Reply:
As someone who's trying to use this example app for the first time, I'd like to know how to make a request to the app.

When I launched this example project, Visual Studio launches the default browser and navigates to https://localhost:53750/, which returns error 404. I had to looked into the code to find out that the app serves traffic on `/WeatherForecast`.

Reply:
Perhaps I could add a link to the tutorial https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-6.0&tabs=visual-studio for how to make a request.

Reply:
these does not belong to Prometheus/Metrics. Its general usage of the app, which should be in same readme, but much earlier.

Reply:
I've added a section title for this line "## How to run the example", and moved it in front of "## Export metrics to Prometheus".